build/azure-pipelines/linux/product-build-linux.yml

```diff
@@ -60,6 +60,21 @@ steps:
     node build/lib/builtInExtensions.js
   displayName: Install distro dependencies and extensions
 
+- script: |
+    set -e
+    cd ..
+    git clone https://github.com/microsoft/vscode-telemetry-extractor.git
+    cd vscode-telemetry-extractor
+    npm install
+    ./node_modules/typescript/bin/tsc
+    npm run setup-extension-repos
+    node ./out/cli-extract.js --sourceDir ../s --excludedDirPattern extensions  --outputDir . --applyEndpoints --includeIsMeasurement --patchWebsiteEvents
+    node ./out/cli-extract-extensions.js --sourceDir ./src/telemetry-sources/ --outputDir . --applyEndpoints --includeIsMeasurement
+    mv declarations-resolved.json ../s/src/telemetry-core.json
```

Comment:
I know I suggested to put the files in `src`, but not knowing the context then I didn't know better. These files should live next to `package.json` in the built product, right here:

![image](https://user-images.githubusercontent.com/22350/60090275-85352880-9742-11e9-8a6b-70da35f1430a.png)

In order to achieve this:

1. Move them into `$BUILD_STAGINGDIRECTORY/telemetry*.json` instead
2. If `process.env['BUILD_STAGINGDIRECTORY']` is set, have a `gulp.src('telemetry*.json', { cwd: process.env['BUILD_STAGINGDIRECTORY'] })` stream piping into the pipeline mentioned above.

Reply:
Okay sounds good, I'll look into this